app/src/main/java/com/orange/ouds/app/ui/components/topappbar/TopAppBarDemoScreen.kt

```diff
@@ -0,0 +1,208 @@
+        )
+        CustomizationSwitchItem(
+            label = stringResource(R.string.app_components_topAppBar_centerAligned_label),
+            checked = centerAligned,
+            onCheckedChange = { centerAligned = it },
+            enabled = centerAlignedSwitchEnabled
+        )
+        CustomizationSwitchItem(
+            label = stringResource(R.string.app_components_topAppBar_navigationIcon_label),
+            checked = navigationIcon,
+            onCheckedChange = { navigationIcon = it }
+        )
+        CustomizationTextField(
+            label = stringResource(R.string.app_components_topAppBar_title_label),
+            value = title,
+            onValueChange = { value -> title = value }
+        )
+        CustomizationFilterChips(
+            applyTopPadding = true,
+            label = stringResource(R.string.app_components_topAppBar_avatar_label),
+            chipLabels = TopAppBarDemoState.Avatar.entries.map { stringResource(it.labelRes) },
+            selectedChipIndex = TopAppBarDemoState.Avatar.entries.indexOf(avatar),
+            onSelectionChange = { id -> avatar = TopAppBarDemoState.Avatar.entries[id] }
+        )
+        CustomizationTextField(
+            label = stringResource(R.string.app_components_topAppBar_avatarMonogram_label),
+            value = avatarMonogram.toString().trim(),
+            onValueChange = { value -> avatarMonogram = value.firstOrNull().orElse { ' ' } },
+            enabled = avatarMonogramTextFieldEnabled
+        )
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun TopAppBarDemoContent(state: TopAppBarDemoState) {
+    with(state) {
+        val navigationIcon = if (navigationIcon) OudsTopAppBarNavigationIcon.Back {} else null
+        val avatarContentDescription = stringResource(R.string.app_components_topAppBar_secondAction_a11y)
+        val avatarAction = when (avatar) {
+            TopAppBarDemoState.Avatar.Image -> OudsTopAppBarAction.Avatar(
+                painter = painterResource(id = R.drawable.il_top_app_bar_avatar),
+                contentDescription = avatarContentDescription,
+                onClick = {}
+            )
+            TopAppBarDemoState.Avatar.Monogram -> OudsTopAppBarAction.Avatar(
+                monogram = avatarMonogram,
+                color = Color.White,
+                backgroundColor = Color(0xff138126),
+                contentDescription = avatarContentDescription,
+                onClick = {}
+            )
+        }
+        val actions = listOf(
+            OudsTopAppBarAction.Icon(
+                painter = painterResource(id = LocalThemeDrawableResources.current.tipsAndTricks),
+                contentDescription = stringResource(R.string.app_components_topAppBar_firstAction_a11y),
+                onClick = {}
+            ),
+            avatarAction
+        )
+        when (size) {
+            TopAppBarDemoState.Size.Small -> {
```

Comment:
There is maybe nothing to do but I'm not able to reach the demo top app bar buttons in keyboard navigation, it's ok when the top app bar is in scaffold top app bar but not when it is in scaffold content.